source/ui-deployment/src/components/wizard/tools-content.jsx

```diff
@@ -26,6 +26,44 @@ export const TOOLS_CONTENT = {
                 }
             ]
         },
+        existingUserPool: {
```

Comment:
This works but we most recently switched to having the relevant tools content for each component be in that components file so we are slowly moving away from having it all in `tools-content.jsx`. See here for an example: https://github.com/aws-solutions/generative-ai-application-builder-on-aws/blob/019a9cd769097326f4881571203f43849685bc5f/source/ui-deployment/src/components/wizard/Prompt/PromptExperience.tsx#L186

Reply:
Thanks @OmarRad , noted.